### ISSUE TITLE
Unkown characters 

### ISSUE THREAD

**Issue by Ahlid** [sjlevine/atom-slime#69]
I'm having this output with a strange squares

![image](https://user-images.githubusercontent.com/11204019/36450389-b5122e2a-1685-11e8-94ec-7bbf5f02c3c7.png)

**Comment by sjlevine** [comment]
Hi! I believe this is related to #66 -- and is caused is caused by the invisible characters currently used by this plugin to force syntax highlighting in the REPL. There's a better solution to prevent this with a newer version of the Atom API, but this package hasn't been modified to use it yet.

**Comment by mister-walter** [comment]
Can confirm that the \x1B and \x1A characters inserted by appendText and appendPresentationMarker are causing this problem. Modifying those functions to disable the addition of those characters fixed the problem for me.

Modified functions (in atom-slime-repl-view.coffee)
```
  # Adds non-user-inputted text to the REPL
  appendText: (text, colorTags=true) ->
    @inputFromUser = false
    #if colorTags
    #  @editor.insertText("\x1B#{text}\x1B")
    #else
    @editor.insertText(text)
    @inputFromUser = true


  appendPresentationMarker: () ->
    @inputFromUser = false
    #@editor.insertText("\x1A")
    @inputFromUser = true
```

**Comment by sjlevine** [comment]
Fixed by #78!
